src/SdFat_Adafruit_Fork.h

```diff
@@ -0,0 +1,6 @@
+#ifndef SDFAT_ADAFRUIT_FORK_H
+#define SDFAT_ADAFRUIT_FORK_H
+
+#include "SdFat.h"
```

Comment:
maybe chaging to use relative path, the reason is to prevent arduino-cli to pul the official SdFat library along with our fork since we use SdFat.h here (which may complicated linking). Though it may be totally unneccessary (not sure) 

```
#include "./SdFat.h"
```